lexico/utils.py

```diff
@@ -3,15 +3,22 @@
 import os
 import json
 import sqlite3
-
 import arrow
 from wordnik import *
 from tabulate import tabulate
-
+import requests
 from .errors import ConfigFileError
 
 API_URL = 'http://api.wordnik.com/v4'
 
+def incorrect_key(API_KEY):
+    check_url='http://api.wordnik.com/v4/word.json/cat/definitions?api_key={}'.format(API_KEY)
+    if(requests.get(check_url).status_code==200):
```

Comment:
We are using [Wordnik's Python API library](https://github.com/wordnik/wordnik-python3).
So, we do not need to make a `request` to the API.

I like the idea that you have used. However, is there any other lightweight API endpoint which can be used to verify the key?

Reply:
swagger use urllib, so i will make use of that, so no extra import will be required.